src/components/gameComponent/gameComponent.js

```diff
@@ -1,37 +1,42 @@
-// import React, { Component } from 'react';
-// import Board from '../boardComponent';
+import React, { Component } from 'react';
+import Board from "../boardComponent/boardComponent";
+import logo from '../../images/game-logo3.png';
 
-// class Game extends Component {
 
-    // constructor(props) {
-    //     super(props) // call super parent class constructor props
-    //     this.state = {show: false} // if true the data will be shown
-    //     // this.toggleDiv = this.toggleDiv.bind(this); // bind this to the toggle div
-    // }
+class Main extends Component {
 
-    // toggleDiv = (e) => {
-    //     e.preventDefault();
-    //     console.log("Start Button Clicked");
-    //     const { show } = this.state;
-    //     this.setState({show: !show}) 
-    //     // this.setState = { show: !show} // change values in state attribute
-    //     console.log(this.state);
-    // }
-//     render() {
-//         return (
-//             <div className="Game">
-//                 {/* <div className="start_game">
-//                     <button onClick={this.toggleDiv} type="button" className="btn btn-outline-primary">Start Game</button> 
-//                 </div> */}
-//                 <div className="Game_Board">
-//                 {/* //based on state variable the board will show */}
-//                     {this.state.show && <Board /> } 
-//                 </div>
-//             </div>
-//         );
-//     }
+        constructor(props) {
+        super(props) // call super parent class constructor props
+        this.state = {
+            show: false // if true the data will be shown
```

Comment:
Comments aren't necessary here. Commenting is helpful when it's not clear WHY something is doing something. In this case, `show: false` seems pretty self-explanatory.